tests/BUILD.bazel

```diff
@@ -35,16 +35,40 @@ gazelle_binary(
     visibility = ["//visibility:private"],
 )
 
+# Each test case
+TESTS = [file[0:-len("/WORKSPACE")] for file in glob(include = ["**/WORKSPACE"])]
+
+# The files for each test case
+TEST_FILES = [glob(include = [test + "/**"]) for test in TESTS]
+
+# The plain source files for each test case
+TEST_DATA_FILES = [[d for d in data if not d.find(".test-") != -1] for data in TEST_FILES]
+
+# Files generated for each test case
+TEST_GEN_FILES_IN = [[d for d in data if d.find(".test-") != -1] for data in TEST_FILES]
+TEST_GEN_FILES_OUT = [[d.replace(".test-", ".") for d in data] for data in TEST_GEN_FILES_IN]
+
+# genrules renaming files with ".test-" prefix to "." prefix.
+[
+    [
+        genrule(
+            name = "rename_%s_%s" % (TESTS[i], j),
+            srcs = [TEST_GEN_FILES_IN[i][j]],
+            outs = [TEST_GEN_FILES_OUT[i][j]],
+            cmd = "cp $< $@",
```

Comment:
Something here is not working on windows where this genrule is either not being generated or is not dropping the `.test-` from the path, I assume because of slashes.

Anyone see anything wrong here? Or have a windows box to debug it more?

Reply:
I gave up and ignored this test on windows for now. I confirmed it is renaming the files but the `.test-*` files are still visible. I assume this is a windows+sandboxing issue?